docs/src/guides/ibexa/fastly.md

```diff
@@ -58,7 +58,7 @@ platform variable:set -e production env:FASTLY_KEY YOUR_ID_HERE
 ## Setup the correct VCL files
 
 There are two VCL files provided as starting points for using Fastly with Ibexa DXP;
-you can find them in `vendor/ezsystems/ezplatform-http-cache-fastly/fastly/*.vcl`.
+you can find them in `vendor/ibexa/fastly/fastly/*.vcl`.
```

Comment:
TODO:

There is 3 files:
- ez_main.vcl
- ez_user_hash.vcl
- snippet_re_enable_shielding.vcl

Unlike the two first, the 3rd file must not be uploaded to "Custom VCL" but its content copy-pasted to "VCL snippets".

Reply:
```suggestion
you can find them in `vendor/ibexa/fastly/fastly/ez_*.vcl`.
A VCL snippet can be found in `vendor/ibexa/fastly/fastly/snippet_re_enable_shielding.vcl`.
```